docs/src/layouts/DefaultLayout.vue

```diff
@@ -4,8 +4,16 @@
       <FdShellBar>
         <FdShellBarGroup position="start">
           <FdShellBarLogo
-            src="/images/logo.png"
-            srcset="/images/logo.png 1x, /images/logo@2x.png 2x"
+            :src="
+              env === 'production'
+                ? '/fundamental-vue/images/logo.png'
+                : '/images/logo.png'
+            "
+            :srcset="
+              env === 'production'
+                ? '/fundamental-vue/images/logo.png 1x, /fundamental-vue/images/logo@2x.png 2x'
+                : '/images/logo.png 1x, /images/logo@2x.png 2x'
+            "
```

Comment:
@ChristianKienle is there a more "vue-y" way to do this? Seems a little messy.

Reply:
Hi Jenna,

Perhaps can be put in the data than template. Something like this, maybe:
```
// get path based on environment variable
import getEnvPath from "@/util/getEnvPath.ts"

data() {
  const logo = getEnvPath("/images/logo.png");
  const logoSet =`${getEnvPath("/images/logo.png")} 1x, ${getEnvPath("/images/logo.png")} 2x`;
      
  return {
    ...
    logo,
    logoSet
  };
}
```

and in the template:
`<FdShellBarLogo :src="logo" :srcset="logoSet" />`

Any alternative opinion, @ChristianKienle?

Reply:
There is actually a third way. We could set the path to the image by using an environment variable. We already do this in the ui-directory:

https://github.com/SAP/fundamental-vue/blob/master/ui/src/config.ts

Every environment variable that starts with `VUE_APP` will be available to the Vue app. So we could have a function that returns `/images/logo.png` by default but returns `VUE_APP_LOGO_PATH_1X` (…) if set.

If you want we can do this change for you after this PR has been merged. You decide. :D

Reply:
I ended up just leveraging the base_url env to create the correct pathing :)

Reply:
Today I learned… thanks!